dev-tools/reconfigurator-cli/tests/output/cmds-target-release-stdout

```diff
@@ -8049,20 +8052,232 @@ to:   blueprint 4713f6c4-e8ba-4a28-87a0-df75ebf7b8b6
+-       AAAA fd00:1122:3344:1::1
++   name: 9ae90740-7fdb-4073-ae43-048f2fca3d69.host          (records: 1)
++       AAAA fd00:1122:3344:102::2c
++   name: 9e2e0774-3cf6-4f75-9a12-92db05c77b81.host          (records: 1)
++       AAAA fd00:1122:3344:103::29
+*   name: _crucible-pantry._tcp                              (records: 3 -> 3)
+-       SRV  port 17000 75b220ba-a0f4-4872-8202-dc7c87f062d0.host.control-plane.oxide.internal
+-       SRV  port 17000 ad6a3a03-8d0f-4504-99a4-cbf73d69b973.host.control-plane.oxide.internal
+-       SRV  port 17000 ba4994a8-23f9-4b1a-a84f-a08d74591389.host.control-plane.oxide.internal
++       SRV  port 17000 156bfcde-e3fa-4abe-a93e-eb4a408b4e5e.host.control-plane.oxide.internal
++       SRV  port 17000 7e83b92d-6b02-47f3-a5ab-125a6bb44e29.host.control-plane.oxide.internal
++       SRV  port 17000 9464c6ed-ffa6-4e88-ae4e-76551d82b2af.host.control-plane.oxide.internal
+*   name: _external-dns._tcp                                 (records: 3 -> 3)
+-       SRV  port  5353 6c3ae381-04f7-41ea-b0ac-74db387dbc3a.host.control-plane.oxide.internal
+-       SRV  port  5353 803bfb63-c246-41db-b0da-d3b87ddfc63d.host.control-plane.oxide.internal
+-       SRV  port  5353 f6ec9c67-946a-4da3-98d5-581f72ce8bf0.host.control-plane.oxide.internal
++       SRV  port  5353 26bdd109-c842-43a9-95cb-15aba9b0832b.host.control-plane.oxide.internal
++       SRV  port  5353 4ab0ec67-b27e-42b5-af22-9117ad11113b.host.control-plane.oxide.internal
++       SRV  port  5353 63ff80e3-ef8d-4186-8240-5ebd3f7d4d82.host.control-plane.oxide.internal
+*   name: _internal-ntp._tcp                                 (records: 3 -> 3)
+-       SRV  port   123 62620961-fc4a-481e-968b-f5acbac0dc63.host.control-plane.oxide.internal
+-       SRV  port   123 6444f8a5-6465-4f0b-a549-1993c113569c.host.control-plane.oxide.internal
+-       SRV  port   123 f10a4fb9-759f-4a65-b25e-5794ad2d07d8.host.control-plane.oxide.internal
++       SRV  port   123 9e2e0774-3cf6-4f75-9a12-92db05c77b81.host.control-plane.oxide.internal
++       SRV  port   123 ba87399e-e9b7-4ee4-8cb7-0032822630e9.host.control-plane.oxide.internal
++       SRV  port   123 e14f91b0-0c41-48a0-919d-e5078d2b89b0.host.control-plane.oxide.internal
+*   name: _nameservice._tcp                                  (records: 3 -> 3)
+-       SRV  port  5353 427ec88f-f467-42fa-9bbb-66a91a36103c.host.control-plane.oxide.internal
+-       SRV  port  5353 99e2f30b-3174-40bf-a78a-90da8abba8ca.host.control-plane.oxide.internal
+-       SRV  port  5353 ea5b4030-b52f-44b2-8d70-45f15f987d01.host.control-plane.oxide.internal
++       SRV  port  5353 698d1d82-0620-4978-93ac-0ba5d40f3da9.host.control-plane.oxide.internal
++       SRV  port  5353 abd27551-4027-4084-8b52-13a575b035b4.host.control-plane.oxide.internal
++       SRV  port  5353 d5fd048a-8786-42d3-938e-820eae95d7f4.host.control-plane.oxide.internal
+*   name: _nexus._tcp                                        (records: 3 -> 3)
```

Comment:
This (L8238 - L8244) is the only bit we really care about in this internal DNS diff: after the handoff (not the handoff _blueprint_, but the handoff itself), we've updated the Nexus internal DNS.